lib/active_admin/views/components/status_tag.rb

```diff
@@ -60,7 +53,7 @@ def convert_to_boolean_status(status)
         case status
         when true, 'true'
           'Yes'
-        when false, 'false', nil
+        when *FALSE_VALUES
```

Comment:
Why is the "false values" list incremented? And why is the "true values" list not incremented accordingly?